tests/validate/xrt_iops_test/src/xrt_api_iops.cpp

```diff
@@ -0,0 +1,240 @@
+#include <iostream>
+#include <iomanip>
+#include <vector>
+#include <chrono>
+#include <thread>
+#include <getopt.h>
```

Comment:
I think these tests should be standard c++ and not use Linux getopt. If you are strongly opposed to iterating argc, argv as is done in all the low level xrt tests (tests/xrt/*), then alternatively boost::program_options can be used, but which in IMO is a little too heavyweight for simply examples like this.

Reply:
I would follow the same way as other low level xrt tests.